tools/slack_tools/collect_prs.py

```diff
@@ -8,13 +8,10 @@
 import sys
 
 import github
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
 
-try:
-    input = raw_input  # Python 2
-except NameError:
-    pass  # Python 3
-
-maintainers = {
+Maintainers = {
```

Comment:
sorry to be annoying - the convention for constants in python is  all caps - ie `MAINTAINERS`

Reply:
haha, not annoying at all.  I thought that request was kinda weird because I read it as Maintainers - makes much more sense w.r.t. python style now I'm not misreading it!